edpop_explorer/record.py

```diff
@@ -206,6 +206,12 @@ class BibliographicalRecord(Record):
     extent: Optional[Field] = None
     size: Optional[Field] = None
     physical_description: Optional[Field] = None
+    bookseller: Optional[Field] = None
+    location: Optional[Field] = None
+    update: Optional[Field] = None
+    image: Optional[Field] = None
+    place_of_printing: Optional[Field] = None
+    style: Optional[Field] = None
```

Comment:
I think that these new fields are too database-specific, although I am in doubt about `place_of_printing`. The point about `place_of_printing` is though that most databases don't make a clear distinction between publishing and printing, so that it will be difficult for us to make this distinction in the normalized version of the records...

Reply:
I will delete it then.